deployments/liqo_chart/values.yaml

```diff
@@ -99,3 +103,5 @@ global:
   configmapName: "liqo-configmap"
   suffix: ""
   version: ""
+  liqodash:
```

Comment:
I think this is not needed

Reply:
yes this line had to be removed with the last change. I'll do it now.